index.html

```diff
@@ -0,0 +1,133 @@
+			</ul>
+		</div>
+	</header>
+
+	<section class="homepage">
+		<div class="hamburger">
+			<div class="bars"></div>
+		</div>
+		<div class="container2">
+			<h1 class="header1">"Hello!"</h1>
+			<h2 class="header2">Health Awareness on the connection between Diabetes and Obesity Programme 2023</h2>
+			<p class="paragraph1">This program is aimed at bringing awareness to the masses, and educate them on how obesity can result in Diabetes Mellitus. A detailed explanation of Diabetes Mellitus and its side effects will be given. This way, thousands of lives are saved by this program, and the rate of occurrence of diabetes mellitus is reduced.</p>
+			<p class="paragraph2">2023.05.25(THU) ~ 2023.05.26(FRI)</p>
+			<p class="paragraph3">@ Abuja International Conference Centre, Abuja, Nigeria</p>
+		</div>
+	</section>
+
+	<section class="mainprogram">
+		<div class="container3">
+			<h2 class="header3">Main Program</h2>
+			<hr>
+			<div class="programs">
+				<ul class="proglist">
+					<li class="list-item3">
+						<img src="images/Screenshot (207).png" alt="">
+						<h2 class="header4">Lecture</h2>
+						<p class="paragraph4">Listen to the speakers from various countries about the messages of sharing and opening</p>
+					</li>
+					<li class="list-item3">
+						<img src="images/Screenshot (208).png" alt="">
+						<h2 class="header4">Exhibition</h2>
+						<p class="paragraph4">Appreciate various exhibitions by diffrent companies and products</p>
+					</li>
+					<li class="list-item3">
+						<img src="images/Screenshot (209).png" alt="">
+						<h2 class="header4">Forum</h2>
+						<p class="paragraph4">Have the time to share your thoughts and opinoin with experts about difrrent topics</p>
+					</li>
+					<li class="list-item3">
+						<img src="images/Screenshot (210).png" alt="">
+						<h2 class="header4">Workshop</h2>
+						<p class="paragraph4">Participate in diffrent workshop from provided by senior engineers</p>
+					</li>
+					<li class="list-item3">
+						<img src="images/Screenshot (211).png" alt="">
+						<h2 class="header4">Networking</h2>
+						<p class="paragraph4">Get opportunities to network with affilaates around the countery also after the conference</p>
+					</li>
+				</ul>
+			</div>
+			<button class="btn" type="button">Join Health Awareness Programme</button>
+			<button class="deskbtn" type="button"><a href="#"> SEE THE WHOLE PROGRAM </a></button>
+		</div>
+	</section>
+
+	<section class="speakers">
+		<div class="container4">
+			<h2 class="header5">Featured Speakers</h2>
+			<hr class="hr3">
+			<div class="program gradient maxHeight" id="programs">
+			</div>
+			<div class="button">
+				<span class="showmore">More</span>
+				<span class="showless d-none">Less</span>
+				<i class="fa fa-chevron-down" aria-hidden="true"></i>
+			</div>
+			</div>
+	</section>
+
+	<section class="partners">
+		<div class="sub-cont2">
+		<h2 class="header9">Partner</h2>
+		<hr class="hr3">
```

Comment:
- [ ] You did a great job 😃 here at adding the line below the heading like the Behance, but I see that the color of this hr is different from the one in the Behance, so please 😊 try to put the correct color be like the Behance
- [ ] Also consider adding the correct color for the heading here as you see it is different from the Behance Design, so let's make it like the Behance 😊

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Behance design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/226687538-bd3b0eb7-936b-44b0-835e-3d94ead94351.png)  |    ![image](https://user-images.githubusercontent.com/83360636/226687500-48afd3ad-9872-4bed-b851-2db8d4673d6c.png)   |